### ISSUE TITLE
Level: Rabbot practice

### ISSUE THREAD

**Issue by GamingInky** [andybalaam/rabbit-escape#482]
Basically introducing rabbots
```
:name=Rabbot Practice
:description=Rabbit vs Rabbot, who will win?
:author_name=GamingInky
:author_url=https://github.com/GamingInky
:hint.1.code={g\Btp\B&&}t\\EBBtng)&U%\qBCa
:hint.2.code=S ZS2WWDh{'W]\/hhhdS<
:hint.3.code=#ddd3jWd3\nhhL#hd@\@mLm@We@dnLLhLN##\d4lLn3w4d##LdLWdh3nddWYvdL\ev
:solution.1.code=my!BzOq&*'NO1
:num_rabbits=1
:num_to_save=1
:rabbit_delay=4
:bash=1
           
         y 
      #### 
 Q         
           
     O     
###########
```

**Comment by andybalaam** [comment]
Nice introductory level.  I think we'll make 20 and release them as a level set.  Then later, for the 1.0 release, we'll re-order all the levels to make sure cool stuff like rabbots and water are near the beginning.

**Comment by andybalaam** [comment]
Released in 0.11
